lookout/style/format/langs/javascript/roles.py

```diff
@@ -1,120 +1,249 @@
-ROLES = [
-    "AnyTypeAnnotation",
-    "ArrowFunctionExpression",
-    "AssignmentExpression",
-    "AssignmentPattern",
-    "AwaitExpression",
-    "BinaryExpression",
-    "BlockStatement",
-    "BooleanLiteral",
-    "BooleanLiteralTypeAnnotation",
-    "BooleanTypeAnnotation",
-    "BreakStatement",
-    "CallExpression",
-    "CatchClause",
-    "ClassBody",
-    "ClassDeclaration",
-    "ClassExpression",
-    "ClassMethod",
-    "ClassProperty",
-    "CommentBlock",
-    "CommentLine",
-    "ConditionalExpression",
-    "ContinueStatement",
-    "DeclareClass",
-    "DeclareFunction",
-    "DeclareModule",
-    "DeclareModuleExports",
-    "DeclareVariable",
-    "Directive",
-    "DirectiveLiteral",
-    "DoWhileStatement",
-    "ExistsTypeAnnotation",
-    "ExportAllDeclaration",
-    "ExportDefaultDeclaration",
-    "ExportNamedDeclaration",
-    "ExportSpecifier",
-    "ExpressionStatement",
-    "File",
-    "ForInStatement",
-    "ForOfStatement",
-    "ForStatement",
-    "FunctionDeclaration",
-    "FunctionExpression",
-    "FunctionTypeAnnotation",
-    "FunctionTypeParam",
-    "GenericTypeAnnotation",
-    "Identifier",
-    "IfStatement",
-    "Import",
-    "ImportDeclaration",
-    "ImportDefaultSpecifier",
-    "ImportNamespaceSpecifier",
-    "ImportSpecifier",
-    "InterfaceExtends",
-    "IntersectionTypeAnnotation",
-    "JSXAttribute",
-    "JSXClosingElement",
-    "JSXElement",
-    "JSXEmptyExpression",
-    "JSXExpressionContainer",
-    "JSXIdentifier",
-    "JSXMemberExpression",
-    "JSXOpeningElement",
-    "JSXSpreadAttribute",
-    "JSXText",
-    "LabeledStatement",
-    "LogicalExpression",
-    "MemberExpression",
-    "MixedTypeAnnotation",
-    "NewExpression",
-    "NullableTypeAnnotation",
-    "NullLiteral",
-    "NullLiteralTypeAnnotation",
-    "NumberLiteralTypeAnnotation",
-    "NumberTypeAnnotation",
-    "NumericLiteral",
-    "ObjectExpression",
-    "ObjectMethod",
-    "ObjectPattern",
-    "ObjectProperty",
-    "ObjectTypeAnnotation",
-    "ObjectTypeIndexer",
-    "ObjectTypeProperty",
-    "OpaqueType",
-    "Program",
-    "RestElement",
-    "ReturnStatement",
-    "SequenceExpression",
-    "SpreadElement",
-    "StringLiteral",
-    "StringLiteralTypeAnnotation",
-    "StringTypeAnnotation",
-    "Super",
-    "SwitchCase",
-    "SwitchStatement",
-    "TaggedTemplateExpression",
-    "TemplateElement",
-    "ThisExpression",
-    "ThrowStatement",
-    "TryStatement",
-    "TupleTypeAnnotation",
-    "TypeAlias",
-    "TypeAnnotation",
-    "TypeCastExpression",
-    "TypeofTypeAnnotation",
-    "TypeParameter",
-    "TypeParameterDeclaration",
-    "TypeParameterInstantiation",
-    "UnaryExpression",
-    "UnionTypeAnnotation",
-    "UpdateExpression",
-    "VariableDeclaration",
-    "VariableDeclarator",
-    "Variance",
-    "VoidTypeAnnotation",
-    "WhileStatement",
-    "YieldExpression",
-]
+ROLES_FREQ = {
```

Comment:
Please do not break the existing format and code.

If you need to save the stats, save it as another list where that values are in the same sequence as the roles list.

Reply:
I do not break anything. https://github.com/src-d/style-analyzer/pull/124/files/2c8bb6b25fa86512786bcf927b7b0474747b9b71#diff-936e060b4c3b7aec01f5ea879ad7f996R246

Reply:
ok